SourceKitStressTester/Tests/SwiftCWrapperToolTests/SwiftCWrapperToolTests.swift

```diff
@@ -76,6 +76,7 @@ class SwiftCWrapperToolTests: XCTestCase {
   }
 
   func testStressTesterOperationQueue() throws {
+    try XCTSkipIf(true, "Failing non-deterministically. Disabling until we have time to investigate - rdar://100970606")
```

Comment:
This test has a bunch of operations that run a bunch of times with a sleep. We wait for the second to complete and then cancel the rest, so not super surprising this is flaky.

We should continue `second` until `first` is finished (could pass some bool down) and just loop `third` and `fourth` forever (until cancelled). If `waitUntilFinished` doesn't have a timeout we should add that as well.

I'm fine with just disabling this for now though, we don't make many/any changes here.